tests/shell/tests.py

```diff
@@ -0,0 +1,15 @@
+from django import __version__
+from django.core.management import call_command
+from django.test import SimpleTestCase
+from django.test.utils import patch_logger
+
+
+class ShellCommandTestCase(SimpleTestCase):
+
+    def test_command_option(self):
+        with patch_logger('test', 'info') as logger:
```

Comment:
I don't think that will work (or at least I couldn't get it to work): the print statement would be ```exec```ed, which would not redirect it to stdout as with normal management command prints. I left it the way it is, unless you can show me a better solution.